Sources/KlaviyoSwift/StateManagement/StateManagement.swift

```diff
@@ -479,6 +488,17 @@ struct KlaviyoReducer: ReducerProtocol {
 
             return .none
 
+        case let .setBadgeCount:
+            if let userDefaults = UserDefaults(suiteName: Bundle.main.object(forInfoDictionaryKey: "Klaviyo_App_Group") as? String) {
```

Comment:
@belleklaviyo were you planning on using the integer that was passed in?

@ndurell the idea was that this method acts as a pseudo-swizzled method for developers to set badge count with instead of using `UNUserNotificationCenter.current().setBadgeCount` and us trying to swizzle it. Of course it also acts as a way to reset badge counts.

Reply:
Maybe should separate to a separate action but either should be moved to an effect.

Reply:
yeah meant to pass in the int, was a typo -- fixed